packages/components/src/molecules/SearchInputField/SearchInputField.tsx

```diff
@@ -68,6 +69,9 @@ const SearchInputField = forwardRef<
     }
   }
 
+  const { 'aria-label': buttonAriaLabel, ...restButtonProps } =
```

Comment:
```suggestion
  const { 'aria-label': buttonAriaLabel, ...otherButtonProps } =
```

nitpick: I think we are using `other` prefix in other cases.